hkmc2/shared/src/main/scala/hkmc2/semantics/Term.scala

```diff
@@ -56,25 +80,48 @@ sealed trait ResolvableImpl:
       case _ => N
     case _ => N
   
+  
+  def callableDefn: Opt[CallableDefinition] = defn.flatMap:
+    case td: TermDefinition => S:
+      CallableDefinition(
+        td.sym,
+        td.params,
+        td.tparams,
+        td.sign,
+        td.flags,
+        td.modulefulness,
+        td,
+      )
+    case td: ClassLikeDef => S:
+      CallableDefinition(
+        td.bsym, 
+        td.paramsOpt.toList ::: td.auxParams, 
+        S(td.tparams.map(tp => Param(FldFlags.empty, tp.sym, N, Modulefulness.none))), 
+        N, // TODO: handle class-like definitions with signatures
+        TermDefFlags.empty, // TODO: handle class-like definitions with flags
+        Modulefulness.none, // TODO: handle modulefulness for class-like definitions
```

Comment:
What does this TODO mean?

Reply:
These TODOs were added earlier when I created the resolver. It's in the change only because I moved the thing from `Resolver.scala` to `Term.scala`. I plan to remove|fix them together with the symbol refactorization.